### PR TITLE
Upgrade to 7.0.0 scarthgap

### PR DESCRIPTION
**Pull request by schmidma**
## Introduced Changes

upgrades to the our latest yocto release

### CODE REVIEW

File: crates/code_generation/src/run.rs [hunk 1]
```diff
@@ -46,6 +46,10 @@ pub fn generate_run_function(cyclers: &Cyclers) -> TokenStream {
 
             #start_cyclers
 
+            #[cfg(feature = "systemd")]
+            systemd::daemon::notify(false, [(systemd::daemon::STATE_READY, "1")].iter())
```
Comment (knoellle) [inline crates/code_generation/src/run.rs:50]: Use [`std::iter::once()`](https://doc.rust-lang.org/std/iter/fn.once.html) here.

File: crates/constants/src/lib.rs [hunk 2]
```diff
@@ -13,8 +13,8 @@ pub const HULA_DBUS_PATH: &str = "/org/hulks/HuLA";
 pub const HULA_DBUS_SERVICE: &str = "org.hulks.hula";
 pub const HULA_SOCKET_PATH: &str = "/tmp/hula";
 pub const OS_RELEASE_PATH: &str = "/etc/os-release";
-pub const OS_VERSION: &str = "5.9.1";
-pub const SDK_VERSION: &str = "5.9.0";
+pub const OS_VERSION: &str = "6.0.0";
+pub const SDK_VERSION: &str = "6.0.0";
```
Comment (knoellle) [inline crates/constants/src/lib.rs:17]: 7.0.0

**Review by knoellle — CHANGES_REQUESTED** [review]

File: crates/constants/src/lib.rs (hunk 2 above)
Comment (schmidma) [reply crates/constants/src/lib.rs:17]: Done

File: crates/code_generation/src/run.rs (hunk 1 above)
Comment (schmidma) [reply crates/code_generation/src/run.rs]: Done, changed it also from my copy source in hula ;)

**Review by knoellle — APPROVED** [review]
LGTM

**Pull request merged**